<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="" lang="">
<head>
  <title>Test Results Report - Firefox_Test5</title>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
  <meta name="description" content="TestNG unit test results." />
  <link href="reportng.css" rel="stylesheet" type="text/css" />
    <script type="text/javascript" src="reportng.js"></script>
</head>
<body>
<h1>Firefox_Test5</h1>
<p>
  Test duration: 5134.168s
</p>



  <table class="resultsTable">
    <tr><th colspan="3" class="header failed">Failed Tests</th></tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_CashEnd_ViewAccount_U17</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    50.157s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-0', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: </b></a><br />
    <div class="stackTrace" id="exception-0">
              org.testng.AssertJUnit.fail(AssertJUnit.java:59)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)<br />
              com.autotest.testcases.YYS008_CashEnd_ViewAccount_U17.test(YYS008_CashEnd_ViewAccount_U17.java:184)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:702)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:768)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:37)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br />
              java.lang.Thread.run(Unknown Source)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_CashEnd_ViewAccount_U18</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    57.159s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-1', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: </b></a><br />
    <div class="stackTrace" id="exception-1">
              org.testng.AssertJUnit.fail(AssertJUnit.java:59)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)<br />
              com.autotest.testcases.YYS008_CashEnd_ViewAccount_U18.test(YYS008_CashEnd_ViewAccount_U18.java:159)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:702)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:768)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:37)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br />
              java.lang.Thread.run(Unknown Source)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_CashEnd_ViewAccount_U21</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    40.578s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-2', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: </b></a><br />
    <div class="stackTrace" id="exception-2">
              org.testng.AssertJUnit.fail(AssertJUnit.java:59)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)<br />
              com.autotest.testcases.YYS008_CashEnd_ViewAccount_U21.test(YYS008_CashEnd_ViewAccount_U21.java:148)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:702)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:768)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:37)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br />
              java.lang.Thread.run(Unknown Source)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_CashEnd_ViewAccount_U24</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    35.847s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-3', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: </b></a><br />
    <div class="stackTrace" id="exception-3">
              org.testng.AssertJUnit.fail(AssertJUnit.java:59)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)<br />
              com.autotest.testcases.YYS008_CashEnd_ViewAccount_U24.test(YYS008_CashEnd_ViewAccount_U24.java:168)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:702)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:768)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:37)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br />
              java.lang.Thread.run(Unknown Source)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_FastCashProject_U17</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    264.565s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-4', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: </b></a><br />
    <div class="stackTrace" id="exception-4">
              org.testng.AssertJUnit.fail(AssertJUnit.java:59)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)<br />
              com.autotest.testcases.YYS008_FastCashProject_U17.test(YYS008_FastCashProject_U17.java:268)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:702)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:768)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:37)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br />
              java.lang.Thread.run(Unknown Source)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_FastCashProject_U21</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    52.993s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-5', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: </b></a><br />
    <div class="stackTrace" id="exception-5">
              org.testng.AssertJUnit.fail(AssertJUnit.java:59)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)<br />
              com.autotest.testcases.YYS008_FastCashProject_U21.test(YYS008_FastCashProject_U21.java:242)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:702)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:768)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:37)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br />
              java.lang.Thread.run(Unknown Source)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_FastCashProject_U24</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    47.760s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-6', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: </b></a><br />
    <div class="stackTrace" id="exception-6">
              org.testng.AssertJUnit.fail(AssertJUnit.java:59)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)<br />
              com.autotest.testcases.YYS008_FastCashProject_U24.test(YYS008_FastCashProject_U24.java:246)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:702)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:768)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:37)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br />
              java.lang.Thread.run(Unknown Source)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_InvestSDT_U17</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    57.201s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-7', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: </b></a><br />
    <div class="stackTrace" id="exception-7">
              org.testng.AssertJUnit.fail(AssertJUnit.java:59)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)<br />
              com.autotest.testcases.YYS008_InvestSDT_U17.test(YYS008_InvestSDT_U17.java:520)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:702)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:768)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:37)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br />
              java.lang.Thread.run(Unknown Source)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_InvestSDT_U21</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    169.950s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-8', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: </b></a><br />
    <div class="stackTrace" id="exception-8">
              org.testng.AssertJUnit.fail(AssertJUnit.java:59)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)<br />
              com.autotest.testcases.YYS008_InvestSDT_U21.test(YYS008_InvestSDT_U21.java:425)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:702)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:768)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:37)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br />
              java.lang.Thread.run(Unknown Source)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_InvestSDT_U24</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    166.134s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-9', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: </b></a><br />
    <div class="stackTrace" id="exception-9">
              org.testng.AssertJUnit.fail(AssertJUnit.java:59)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)<br />
              com.autotest.testcases.YYS008_InvestSDT_U24.test(YYS008_InvestSDT_U24.java:405)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:702)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:768)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:37)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br />
              java.lang.Thread.run(Unknown Source)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_PayMent</td>
      </tr>
            
<tr>
  <td class="method">
                  Test
      </td>
  <td class="duration">
    550.090s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-10', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: </b></a><br />
    <div class="stackTrace" id="exception-10">
              org.testng.AssertJUnit.fail(AssertJUnit.java:59)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)<br />
              com.autotest.testcases.YYS008_PayMent.Test(YYS008_PayMent.java:348)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:702)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:768)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:37)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br />
              java.lang.Thread.run(Unknown Source)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_Repay_Business</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    464.664s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-11', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: </b></a><br />
    <div class="stackTrace" id="exception-11">
              org.testng.AssertJUnit.fail(AssertJUnit.java:59)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)<br />
              com.autotest.testcases.YYS008_Repay_Business.test(YYS008_Repay_Business.java:293)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:702)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:768)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:37)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br />
              java.lang.Thread.run(Unknown Source)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_RewardAdd</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    12.715s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-12', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: </b></a><br />
    <div class="stackTrace" id="exception-12">
              org.testng.AssertJUnit.fail(AssertJUnit.java:59)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)<br />
              com.autotest.testcases.YYS008_RewardAdd.test(YYS008_RewardAdd.java:153)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:702)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:768)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:37)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br />
              java.lang.Thread.run(Unknown Source)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_UserInvest</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    582.953s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-13', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: </b></a><br />
    <div class="stackTrace" id="exception-13">
              org.testng.AssertJUnit.fail(AssertJUnit.java:59)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)<br />
              org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)<br />
              com.autotest.testcases.YYS008_UserInvest.test(YYS008_UserInvest.java:382)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:702)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:768)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:37)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br />
              java.lang.Thread.run(Unknown Source)<br />
                </div>
      </td>
</tr>
      </table>


  <table class="resultsTable">
    <tr><th colspan="3" class="header passed">Passed Tests</th></tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.Project_Release_YYS008</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    419.212s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.UserRegistration</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    1604.925s
  </td>
  <td class="result">
        
          <i>Method arguments: </i><span class="arguments">"11", "17"</span><br />
    
        

      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.autotest.testcases.YYS008_GenPayPlan</td>
      </tr>
            
<tr>
  <td class="method">
                  test
      </td>
  <td class="duration">
    321.727s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
      </table>
  
</body>
</html>
